app/src/main/kotlin/co/netguru/baby/monitor/client/feature/communication/webrtc/client/RtcClientController.kt

```diff
@@ -0,0 +1,65 @@
+package co.netguru.baby.monitor.client.feature.communication.webrtc.client
+
+import android.content.Context
+import co.netguru.baby.monitor.client.common.view.CustomSurfaceViewRenderer
+import co.netguru.baby.monitor.client.feature.communication.webrtc.StreamState
+import co.netguru.baby.monitor.client.feature.communication.websocket.MessageParser
+import co.netguru.baby.monitor.client.feature.communication.websocket.RxWebSocketClient
+import io.reactivex.disposables.CompositeDisposable
+import io.reactivex.rxkotlin.addTo
+import io.reactivex.rxkotlin.subscribeBy
+import io.reactivex.schedulers.Schedulers
+import timber.log.Timber
+import java.net.URI
+import javax.inject.Inject
+
+class RtcClientController @Inject constructor(
+    private val messageParser: MessageParser
+) {
+    private var rtcClient: RtcClient? = null
+    private val compositeDisposable = CompositeDisposable()
+
+    fun startCall(
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/9505a80fe42460a19170990c8c0b687614740b84/app/src/main/kotlin/co/netguru/baby/monitor/client/feature/communication/webrtc/client/RtcClientController.kt#L22"></span>The function startCall has too many parameters. The current threshold is set to 6.</td>
    </tr>
  </tbody>
</table>